### PR TITLE
chore: update tests to use blueprint test framework

### PR DESCRIPTION
**Pull request by bharathkkb**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: build/int.cloudbuild.yaml [hunk 1]
```diff
@@ -23,16 +23,16 @@ steps:
   - 'TF_VAR_billing_account=$_BILLING_ACCOUNT'
 - id: create
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
-  args: ['/bin/bash', '-c', 'cft test run all --stage init --verbose']
+  args: ['/bin/bash', '-c', 'cft test run all --stage init --verbose --test-dir test/integration']
```
Comment (bharathkkb) [inline build/int.cloudbuild.yaml:26]: Workaround until https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/1082

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: build/int.cloudbuild.yaml [hunk 2]
```diff
@@ -23,19 +23,19 @@ steps:
   - 'TF_VAR_billing_account=$_BILLING_ACCOUNT'
 - id: create
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
-  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do create']
+  args: ['/bin/bash', '-c', 'cft test run all --stage init --verbose --test-dir test/integration']
```
Comment (apeabody) [inline build/int.cloudbuild.yaml:26]:
```suggestion
  args: ['/bin/bash', '-c', 'cft test run all --stage init --verbose']
```

File: build/int.cloudbuild.yaml [hunk 3]
```diff
@@ -23,19 +23,19 @@ steps:
...
 - id: converge
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
-  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do converge']
+  args: ['/bin/bash', '-c', 'cft test run all --stage apply --verbose --test-dir test/integration']
```
Comment (apeabody) [inline build/int.cloudbuild.yaml:29]:
```suggestion
  args: ['/bin/bash', '-c', 'cft test run all --stage apply --verbose']
```

File: build/int.cloudbuild.yaml [hunk 4]
```diff
@@ -23,19 +23,19 @@ steps:
...
 - id: verify
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
-  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do verify']
+  args: ['/bin/bash', '-c', 'cft test run all --stage verify --verbose --test-dir test/integration']
 - id: destroy
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
-  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && kitchen_do destroy']
+  args: ['/bin/bash', '-c', 'cft test run all --stage teardown --verbose --test-dir test/integration']
```
Comment (apeabody) [inline build/int.cloudbuild.yaml:35]:
```suggestion
  args: ['/bin/bash', '-c', 'cft test run all --stage teardown --verbose']
```

**Review by apeabody — APPROVED** [review]

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@bharathkkb
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**